versions.gradle

```diff
@@ -16,11 +16,11 @@ ext {
             kotshi          : "0.3.0",
             archLifecycle   : "1.1.0",
             archRoom        : "1.0.0",
-            dagger          : "2.13",
-            firebase        : "11.6.2", // Adjust version for emulator
+            dagger          : "2.14.1",
+            firebase        : "11.8.0", // Adjust version for emulator
```

Comment:
I want to use this version.11.6.2 Because emulator does not run 😭

Reply:
Is your emulator the favorite feature working?

Reply:
Ah, you are right, favourite is not working :eyes:  The logcat says
```
FavoriteFirestoreDatabase$NotPreparedException
```

I'll check

Reply:
🙇

Reply:
Hmm, even on the `master` branch I get the same exception. I'll create a bug ticket for it

Reply:
I think it is related https://github.com/DroidKaigi/conference-app-2018/issues/277
Did you change device time? 🤔

Reply:
I didn't change device time... should I try that?

Reply:
No, You don't have to try it!
Is Your emulator Google API or Google Play?
![image](https://user-images.githubusercontent.com/1386930/35558120-864345da-05ea-11e8-815d-0cc266c11f0d.png)

Reply:
It contains Google Play Services

Reply:
it was Google API... :bow:

Reply:
It works for me now if I use Google Play emulator, is it still not working for you?

Reply:
I think google apis is okay.
Has the emulator version been updated?